clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -115,6 +115,16 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
         const bestTranslation = selectBestNoticeTranslation(i18n, notice);
         return { notice: { ...notice, disabled }, bestTranslation };
       });
+
+      const noticeOnly = items.filter(
+        (item) =>
+          item.notice.consent_mechanism === ConsentMechanism.NOTICE_ONLY,
+      );
+      const others = items.filter(
+        (item) =>
+          item.notice.consent_mechanism !== ConsentMechanism.NOTICE_ONLY,
+      );
+      return [...noticeOnly, ...others];
```

Comment:
This looks and works great, but I wonder if it wouldn't be better to have this happen in the init stage before the `consent` object gets set on Fides and in the cookie so that the ordering matches there also? Not a huge deal, just a nit-picky suggestion.